perception/autoware_tensorrt_yolox/src/tensorrt_yolox_node.cpp

```diff
@@ -128,6 +133,11 @@ void TrtYoloXNode::onConnect()
 
 void TrtYoloXNode::onImage(const sensor_msgs::msg::Image::ConstSharedPtr msg)
 {
+  if (!setCudaDeviceId(gpu_id_)) {
+    RCLCPP_ERROR(this->get_logger(), "GPU %d does not exist or is not suitable.", gpu_id_);
```

Comment:
nits: How about using `RCLCPP_ERROR_THROTTLE` ?

Reply:
It has been updated, thanks